style.css

```diff
@@ -3,51 +3,62 @@ body {
   font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
 }
 
```

Comment:
- [ ]  Why don't you use the correct font as the `font-family` you have here is quite different from the one in the template. The template has `font-family: Roboto;` and not `font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;`. I can see that you have the correct font family in the head of your `HTML` so how you are going to use it in your `CSS` is like this `font-family: 'Roboto', sans-serif;`

Reply:
Thank you, really did a big mistake,  I changed my font-family to `font-family: 'Roboto', sans-serif`